packages/grafana-ui/rollup.config.ts

```diff
@@ -43,7 +43,7 @@ const buildCjsPackage = ({ env }) => {
         // When 'rollup-plugin-commonjs' fails to properly convert the CommonJS modules to ES6 one has to manually name the exports
         // https://github.com/rollup/rollup-plugin-commonjs#custom-named-exports
         namedExports: {
-          '../../node_modules/lodash/lodash.js': [
+          'node_modules/lodash/lodash.js': [
```

Comment:
doesn't this break grafana/ui build?

Reply:
Before I made this change the build was failing at the `grafana/ui` build step with `Error: 'isString' is not exported by node_modules/lodash/lodash.js, imported by compiled/components/TimePicker/TimeZonePicker.js`. So I updated the path in rollup config to match the location of lodash mentioned in the error.

Digging a little deeper into the grafana-ui bundle output this branch introduces duplicates of lodash.

| Before | After |
| --- | --- |
| <img width="1470" alt="before" src="https://user-images.githubusercontent.com/73201/99534202-53f87080-29a7-11eb-8ed1-77cd002a941d.png"> | <img width="1577" alt="after" src="https://user-images.githubusercontent.com/73201/99534209-56f36100-29a7-11eb-99ca-99d25bebcd35.png"> |

If anyone has the time I would appreciate some input on how best to resolve this prior to merging.

Reply:
@jackw I think the duplicate is caused by direct lodash imports used in grafana/ui. So if there is an `import ... from 'lodash'` instead of `import X from 'lodash/X'` then the entire lodash.js will be added to the bundle.

Reply:
And it seems like there's plenty of such imports in grafana-ui:
![image](https://user-images.githubusercontent.com/2376619/99535186-d6356480-29a8-11eb-9845-5da798fe3b21.png)

Reply:
Yep, tree shaking Lodash requires some effort on our part but if you look at the bundle visualisations you'll see lodash has moved in to the node_modules of grafana/ui in this branch and remains in the root node_modules too (due to it being used by our other deps) which then causes the bundle size to increase. Maybe I create an issue to clean this up after merging this PR? Also what would be the best way to confirm grafana/ui is still functioning as expected?

Reply:
> Maybe I create an issue to clean this up after merging this PR?

Yes, this is a good idea to look at it separately.

> Also what would be the best way to confirm grafana/ui is still functioning as expected?

First check if it builds correctly with the updated config. Then you can test it against panel plugin template from toolkit and verify it works. You can follow this guide https://github.com/grafana/grafana/blob/master/packages/README.md#setting-up-grafana-packages-for-local-development to use the built grafana-ui locally.

Reply:
I finally got round to testing the bundle locally following your instructions @dprokop and then going through the various "build a plugin" tutorials. I also pulled it into an existing metrics plugin and all looks good. I'll create an issue shortly for the lodash cleanup.

Reply:
Yeah, the same happens also with Prism and it is breaking Intellisense (as we are relying on global Prism object that should be same in grafana/ui and core). I have created a fix [here](https://github.com/grafana/grafana/pull/29381) for our local data source, but other data source plugins will be affected by this as well (e.g. X-Ray, Splunk,...)
@zoltanbedi has looked at this as well and when he discovered that when prismjs is removed from grafana/ui's node_module folder it worked again.